lib/services/firewall.pm

```diff
@@ -33,7 +33,9 @@ sub susefirewall2_to_firewalld {
     my $timeout = 180;
     script_run('susefirewall2-to-firewalld -c',                                     timeout => $timeout);
     script_run('firewall-cmd --permanent --zone=external --add-service=vnc-server', timeout => $timeout);
```

Comment:
Suggest change the two lined to assert_script_run since if these cmd failed then later test is no meaning any more.

Reply:
OK, and will rerun to verify it.

Reply:
https://openqa.nue.suse.com/tests/5322341#step/check_upgraded_service/234
https://openqa.nue.suse.com/tests/5322340#step/install_service/421

ALL PASSED